js/main.js

```diff
@@ -0,0 +1,99 @@
+$(document).ready(function() {
+ $.ajax({
+   url: 'http://localhost:3000/articles',
+   type: 'get'
+    dataType: 'json'
+ }).done(ArticleApp.getArticles);
+});
+
+// Create a namespace for my app
+ ArticleApp = ArticleApp || {};
+
+// callback handler for all articles from server
+ArticleApp.getArticles = function(data){
+ //debugger;
+ console.log(data);
+
+ var string = JSON.stringify(data);
+
+
+ // Iterate over all the articles returned from the
```

Comment:
So you got your data, what tools do we use to iterate over a collection? Were those working for you? If not, what did you refer to to make that happen.